hoodie-spark/src/main/java/com/uber/hoodie/DataSourceUtils.java

```diff
@@ -149,7 +149,7 @@ public static HoodieWriteClient createHoodieClient(JavaSparkContext jssc, String
         // override above with Hoodie configs specified as options.
         .withProps(parameters).build();
 
-    return new HoodieWriteClient<>(jssc, writeConfig);
+    return new HoodieWriteClient<>(jssc, writeConfig, true);
```

Comment:
If a consumer wants to perform write() and read() operations in parallel, we have to turn off rollback for one of the operations. Can we make this configurable for now and then address how multiple write clients created for the same dataset should be handled in the longer term later..

Reply:
@n3nash : IIRC, the DataSource read() path does not instantiate HoodieWriteClient. If that is the case, then the change would be consistent between DeltaStreamer and Spark Source. Let me know if this is ok ?

Reply:
sorry my bad, I meant write() and clean() operation..

Reply:
I think the explicit constructor is better here.. user should not have to care about this flag..

Reply:
I agree with explicit constructor, I'm merely saying not hardcoded to true, instead default=true and driven by a config..

Reply:
Just realized, there is no way to trigger explicit clean from data sources I guess, so this is fine..